conans/client/cmd/uploader.py

```diff
@@ -233,7 +232,7 @@ def upload_recipe(self, ref, bundle, remote):
         self._app.remote_manager.upload_recipe(ref, cache_files, remote)
 
         duration = time.time() - t1
-        log_recipe_upload(ref, duration, cache_files, remote.name)
+        self._output.debug(f"Upload {ref} in {duration} time")
```

Comment:
These log traces were actually crashing, trying to compute the md5 checksum of files that didn't exist (broken symlinks)